docs/source/customizations/coprocessor.mdx

```diff
@@ -92,6 +92,20 @@ coprocessor:
 
 In this case, the `RouterService` only sends a coprocessor request whenever it receives a client request. The coprocessor request body includes _no_ data related to the client request (only "control" data, which is [covered below](#coprocessor-request-format)).
 
+### Client configuration
+
+You can control the coprocessor client configuration as needed.  At the moment only experimental_http2 is supported.
+
+For example to enable h2c (http2 cleartext) communication:
```

Comment:
```suggestion
For example, to enable h2c communication with a coprocessor you can use this configuration:
```

Reply:
Applied via 305a2b53ff8a1e918194f86ff0687235aa82b4a6.  Somehow all suggestions were duplicated.  I tried to accept the right ones. ;)